index.html

```diff
@@ -0,0 +1,44 @@
+<!html>
```

Comment:
This file is just an example of the current code in action. The CSS is super-primitive. It's just enough to show and hide the dropdown.

Also, since only the single-select widget is implemented in the current version, that's all that's on display here.